src/saga/adaptors/lsf/lsfjob.py

```diff
@@ -243,7 +256,9 @@ def _lsfcript_generator(url, logger, jd, ppn, lsf_version, queue, span):
                           saga.job.WORKING_DIRECTORY,
                           saga.job.SPMD_VARIATION, # TODO: 'hot'-fix for BigJob
                           saga.job.PROCESSES_PER_HOST,
-                          saga.job.TOTAL_CPU_COUNT],
+                          saga.job.TOTAL_CPU_COUNT,
+                          saga.job.TOTAL_NODE_COUNT,
```

Comment:
For other adaptors we use the cores per node count to calculate the number of nodes needed (cobalt, torque).  This means we have to configure or hardcode that value - but I'd rater see that in sync with the other adaptors than changing the API.  For the SAGA layer, API uniformity over different backends is one of the defining features.  Would that be possible to approach in the same way as in those adaptors?

Reply:
So `processes_per_host` is the job description entry that allows what you are referring to?

Is this because some resources require to tell them how many MPI processes are going to be submitted per node, regardless if they are going to launch any number of threads?

Reply:
Well, ideally we would get the number of cores/node from runtime inspection, like [here](https://github.com/radical-cybertools/saga-python/blob/devel/src/saga/adaptors/slurm/slurm_job.py#L379).  Alternatively we can hardcode that value in the adaptor, similar to [here](https://github.com/radical-cybertools/saga-python/blob/devel/src/saga/adaptors/torque/torquejob.py#L312) - we hardcode many things already, and should eventually switch to resource configs on the SAGA layer - but that's a different topic altogether.   I would  like to avoid a situation where the users need to know the core count in order to be able to request a number of cores, so a fallback value for when PPN is not set would really be useful.

Reply:
Okay, I see your point. I will add a Summit specific hard coded value for now and mark it with a `TODO`.